examples/platform_channel/lib/main.dart

```diff
@@ -86,5 +86,12 @@ class _PlatformChannelState extends State<PlatformChannel> {
 }
 
 void main() {
-  runApp(const MaterialApp(home: PlatformChannel()));
+  runApp(MaterialApp(
+    title: 'Flutter platform_channel',
+    theme: ThemeData(
+      primarySwatch: Colors.grey,
```

Comment:
I think maybe we can remove `primarySwatch` here. This color property will be gradually phased out ([issue link](https://github.com/flutter/flutter/issues/91772)).